pkgs/applications/science/electronics/horizon-eda/default.nix

```diff
@@ -1,22 +1,25 @@
 { stdenv
+, lib
+, buildPythonPackage ? false
```

Comment:
Not sure if it's a good name, since `python3.pkgs.buildPythonPackage` exists.

Reply:
`buildPythonModule`?